src/main/java/jenkins/scm/api/mixin/ChangeRequest.java

```diff
@@ -0,0 +1,20 @@
+package jenkins.scm.api.mixin;
+
+import org.kohsuke.stapler.export.Exported;
+
+import javax.annotation.Nonnull;
+
+/**
+ * The interface carry with the information of a change request,
+ *  such as pull request or merge request.
+ * @since TODO
+ */
+public interface ChangeRequest {
+    /**
+     * Returns the title of the change request
+     * @return title of the change request
+     */
+    @Exported
```

Comment:
I do not think `@Exported` does anything on an interface method. Could look it up in Stapler, but probably it only checked on the implementing method. https://javadoc.jenkins.io/component/stapler/org/kohsuke/stapler/export/Exported.html shows it is not `@Inherited`, and anyway https://docs.oracle.com/javase/8/docs/api/java/lang/annotation/Inherited.html that would not be checked on implemented interfaces.